src/Unosquare.RaspberryIO.Peripherals/Extensions.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace Unosquare.RaspberryIO.Peripherals
+{
+    using System;
+
+    /// <summary>
+    /// Encapsulates math operations for specific purposes.
+    /// </summary>
+    internal static class Extensions
+    {
+        /// <summary>
+        /// Get distance between two points.
+        /// </summary>
+        /// <param name="a"> Point A. </param>
+        /// <param name="b"> Point B. </param>
+        /// <returns> Distance between two points. </returns>
+        internal static double DistanceTo(this double a, double b) =>
```

Comment:
I'm not sure if this could be a general extension for doubles.

Reply:
eliminated as extension for doubles